Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.16-alpine3.13 AS builder
```

Comment:
Can you use Alpine Linux v3.16 here for consistency?
Also, we could probably change this to Go v1.18. No worries if that's too much of a lift since we'll update to Go v1.19 soon anyways.

Reply:
Hi, apologies for the delay. I am working on this now.

Reply:
I've upgraded the image and golang versions to golang 1.18 and alpine 3.16, and also re-ran go mod tidy to clean up the older modules.

Tests worked well locally, though I did only run test-short. Please let me know if you find any problems.

Reply:
@dhui did you find any problems or we are good to merge it?

Reply:
Sorry for the confusion. I have updated `migrate` to support Go 1.19 since my original comment. Could you use that instead?